components/src/dynamo/vllm/handlers.py

```
@@ -1542,6 +1542,74 @@ def __init__(self, d: dict) -> None:
+                nodes.append(
+                    {
+                        "node_ip": node.get("NodeManagerAddress"),
+                        "total_gpus": node_gpus,
+                    }
+                )
+            available_gpus = float(ray.available_resources().get("GPU", 0.0))
+            result["total_gpus"] = total_gpus
+            result["available_gpus"] = available_gpus
+            result["used_gpus"] = total_gpus - available_gpus
+            result["nodes"] = nodes
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)